src/message.lisp

```diff
@@ -117,20 +122,17 @@
 ;;              `(:reply_to_message_id ,reply-to-message-id)))))
 ;;     (make-request bot "sendMessage" options)))
 
+(defun forward-message (bot chat from-chat message &key disable-notification)
+  "https://core.telegram.org/bots/api#forwardmessage"
+  (let ((options
+         (list :|chat_id| (get-chat-id chat)
+               :|from_chat_id| (get-chat-id from-chat)
+               :|message_id| (get-message-id message))))
+    (when disable-notification (nconc options `((:|disable_notification| . ,disable-notification))))
```

Comment:
This template ``` `((:|disable_notification| . ,disable-notification))``` should be replaced with list as well, because `NCONC` might modify it.

Reply:
You can replace it with `(list (cons :|disable_notification| disable-notification)`. Byt the way, probably there is an error in the existing code, because you are adding `(:|disable_notification| . ,disable-notification)` as CONS into the PLIST (not ALIST):

```lisp
CL-USER> (let ((options (list :chat-id 1 :from 2)))
           (nconc options '((:disable . t)))
           options)
(:CHAT-ID 1 :FROM 2 (:DISABLE . T))
```

It is better to use APPEND instead of NCONC, using this pattern:

```lisp
CL-USER> (let* ((disable t)
                (options (append (list :chat-id 1 :from 2)
                                 (when disable
                                   (list :disable t)))))
           options)
(:CHAT-ID 1 :FROM 2 :DISABLE T)
```

Reply:
Done in 17a1966478bb5d83c10721bd8f1ce655c15b7b85.

Reply:
Great! Thank you!